python/test/unit/tools/test_aot.py

```diff
@@ -111,23 +111,59 @@ def select_compiler():
     return cxx
 
 
+def _cxx_compile_cmd(cxx: str, src: list, include_dirs: list, only_compile=True) -> list:
```

Comment:
Nit: for consistency:
```suggestion
def _cxx_compile_cmd(cxx: str, src: list, include_dirs: list, only_compile: bool = True) -> list:
```